pkg/helper/validate/validate.go

```diff
@@ -15,15 +15,21 @@
 package validate
 
 import (
-	k8s "k8s.io/client-go/kubernetes"
+	"fmt"
 
+	"github.com/tektoncd/cli/pkg/cli"
```

Comment:
nit: The only problem I got with that is the dependency on the `cli` repository :angel: 
Maybe having a interface here would be… better :angel: 

```go
type params interface {
    Clients(…) (…, error)
    Namespaces() string 
}

func NamespaceExists(p params) error {
// …
```

Reply:
The issue I'm running into with an interface is how to include `*Clients` as the return type.

Reply:
Pushed something up for this so cli dependency isn't included.